tests/functional/aws-node-sdk/test/quota/tooling.js

```diff
@@ -0,0 +1,48 @@
+const fetch = require('node-fetch');
+const AWS = require('aws-sdk');
+const xml2js = require('xml2js');
+
+const sendRequest = async (method, host, path, body = '', callback) => {
+    const service = 's3';
+    const endpoint = new AWS.Endpoint(host);
+
+    const request = new AWS.HttpRequest(endpoint);
+    request.method = method.toUpperCase();
+    request.path = path;
+    request.body = body;
+    request.headers.Host = host;
+    request.headers['X-Amz-Date'] = new Date().toISOString().replace(/[:\-]|\.\d{3}/g, '');
+    const sha256hash = AWS.util.crypto.sha256(request.body || '', 'hex');
+    request.headers['X-Amz-Content-SHA256'] = sha256hash;
+    request.region = 'us-east-1';
+
+    const signer = new AWS.Signers.V4(request, service);
+    signer.addAuthorization(AWS.config.credentials, new Date());
+
+    const url = `http://${host}${path}`;
+    const options = {
+        method: request.method,
+        headers: request.headers
+    };
+
+    if (method !== 'GET' && method !== 'HEAD') {
```

Comment:
Do we support HEAD?

Reply:
This was just for testing , going to remove the HEAD method